tests/unit/data_access/test_entity_queries.py

```diff
@@ -13,27 +13,44 @@ def test__apply_limit_and_pagination_filters_with_no_filters_applied():
 
 
 @pytest.mark.parametrize(
-    "period, expected_count, end_date",
+    "period, end_date",
     [
-        (["current"], 1, None),  # Test case for current entities
-        (["historical"], 1, "2023-01-01"),  # Past date for historical entities
-        (["all"], 1, None),  # Test case for no filter
+        (["current"], "2024-12-12"),  # Test case for current entities
```

Comment:
i would recommend changing the 'current' end date to be further in the future (1000 years) otherwise you will have a new job on new years day :stuck_out_tongue: